js/models/conversations.js

```diff
@@ -1035,7 +1035,11 @@
           direction: 'incoming',
           status: ['pending', 'expired'],
         });
-        window.libloki.api.sendBackgroundMessage(this.id);
+        window.libloki.api.sendBackgroundMessage(
+          this.id,
+          window.textsecure.OutgoingMessage.DebugMessageType
+            .INCOMING_FR_ACCEPTED
```

Comment:
add a 'reason' for that background message.
Used to log the message type being sent and better follow message logic